static/app-conf.json

```diff
@@ -85,6 +85,16 @@
       }
     },
 
+    {
+      "type": "XYZ",
+      "name": "OpenTopoMap",
+      "url": "https://tile.opentopomap.org/{z}/{x}/{y}.png",
+      "attributions": "Map data: <a href=\"https://openstreetmap.org/copyright\">©OpenStreetMap</a>-contributors, SRTM | Map representation (Kartendarstellung): © <a href=\"http://opentopomap.org/\">OpenTopoMap</a> (<a href=\"https://creativecommons.org/licenses/by-sa/3.0/\">CC-BY-SA</a>)",
+      "lid": "opentopomap",
+      "displayInLayerList": true, 
+      "visible": true
```

Comment:
I'd prefer to have the OpenTopoMap layer not visible per default since it overlays the OSM base map. What do you think?

Reply:
yes, sure.